Code/Engine/Core/World/Implementation/GameObject.cpp

```diff
@@ -13,7 +13,6 @@ namespace
   {
     ezVariantArray value(ezStaticAllocatorWrapper::GetAllocator());
     value.PushBack(ezStringView("CastShadow"));
-    value.PushBack(ezStringView("AutoColMesh")); // TODO: keep this ?
```

Comment:
This IS used by the greyboxing geometry to decide whether it should generate a collider or not.

Reply:
Ok, saw the change below .... and how updates the documentation? http://ezengine.net/pages/docs/scenes/greyboxing.html

Reply:
https://github.com/ezEngine/docs-src/pull/2